tools/go.mod

```diff
@@ -4,5 +4,7 @@ go 1.13
 
 require (
 	github.com/go-bindata/go-bindata/v3 v3.1.3
+	github.com/gobuffalo/flect v0.2.2 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
```

Comment:
I'm not sure how these two ended up in here but I'd recommend removing these lines. @jimmidyson you have a hunch what might have caused these to be added?

Reply:
agreed to remove it from the PR because it is an independent issue.